src/ReactWrapper.jsx

```diff
@@ -208,7 +208,10 @@ class ReactWrapper {
       throw new Error('ReactWrapper::unmount() can only be called on the root');
     }
     this.single('unmount', () => {
-      this.component.setState({ mount: false });
+      this.component.setState({
+        mount: false,
+        lastRender: this.component._reactInternalInstance._renderedComponent.getPublicInstance(),
```

Comment:
i'm concerned that this could overwrite `this.state.lastRender` inside the component. Shouldn't we find a place to store this info that the component code can't interact with?